workspace/python/ds-pack/2.1.0-base-python-3.10.13/README.md

```diff
@@ -24,7 +24,7 @@ nano           7.2
 vim            9.0
 fish           3.6.0
 zsh            5.9
-R              4.2.0
+R              4.3.1
```

Comment:
Hey Jitesh, its `4.2.2` right?

Reply:
Hey no thats the version of `r-base` package 😅 . The version of `R` installed in `4.3.1` :) .